src/network/proxy.py

```diff
@@ -138,5 +143,5 @@ def handle_connect(self):
 
     def state_proxy_handshake_done(self):
         """Handshake is complete at this point"""
-        self.connectedAt = time.time()
+        self.connectedAt = time.time()      # pylint: disable=attribute-defined-outside-init
```

Comment:
This can be refactored like in `tcp.py` (initialised to `0`).

Reply:
Yeah. That's good idea. (Y)